File: Web.Admin/src/api/services/Auth/login/models/LoginTenantOutput.ts
import type { CommonStatusEnum } from "@/api/enums/CommonStatusEnum";
import type { EditionEnum } from "@/api/enums/EditionEnum";
import type { UserTypeEnum } from "@/api/enums/UserTypeEnum";

/**
 * 登录租户输出
 */
export interface LoginTenantOutput {
	/**
	 * 用户Key
	 */
	userKey?: string;
	/**
	 * 租户名称
	 */
	tenantName?: string;
	/**
	 * 租户简称
	 */
	shortName?: string;
	/**
	 * 租户英文名称
	 */
	spellName?: string;
	/**
	 * 
	 */
	edition?: EditionEnum;
	/**
	 * Logo URL
	 */
	logoUrl?: string;
	/**
	 * 工号
	 */
	employeeNo?: string;
	/**
	 * 姓名
	 */
	employeeName?: string;
	/**
	 * 证件照
	 */
	idPhoto?: string;
	/**
	 * 部门Id
	 */
	departmentId?: number;
	/**
	 * 部门名称
	 */
	departmentName?: string;
	/**
	 * 
	 */
	userType?: UserTypeEnum;
	/**
	 * 
	 */
	status?: CommonStatusEnum;
}

File: Api.Server/src/Center.Service/Login/Dto/LoginTenantOutput.cs
// Copyright © 2018-Present 小方
// SPDX-License-Identifier: Apache-2.0
// 
// 本文件依据 Apache License 2.0 授权，完整条款见仓库根目录 LICENSE。
// 本软件按“原样”提供，相关免责声明及责任限制以许可证及适用法律为准。
// 版权来源、合法使用与二次开发责任说明见仓库根目录 README.md。

namespace Fast.Center.Service.Login.Dto;

/// <summary>
/// 登录租户输出
/// </summary>
public class LoginTenantOutput
{
    /// <summary>
    /// 用户Key
    /// </summary>
    public string UserKey { get; set; }

    /// <summary>
    /// 租户名称
    /// </summary>
    public string TenantName { get; set; }

    /// <summary>
    /// 租户简称
    /// </summary>
    public string ShortName { get; set; }

    /// <summary>
    /// 租户英文名称
    /// </summary>
    public string SpellName { get; set; }

    /// <summary>
    /// 版本
    /// </summary>
    public EditionEnum Edition { get; set; }

    /// <summary>
    /// Logo URL
    /// </summary>
    public string LogoUrl { get; set; }

    /// <summary>
    /// 工号
    /// </summary>
    public string EmployeeNo { get; set; }

    /// <summary>
    /// 姓名
    /// </summary>
    public string EmployeeName { get; set; }

    /// <summary>
    /// 证件照
    /// </summary>
    public string IdPhoto { get; set; }

    /// <summary>
    /// 部门Id
    /// </summary>
    public long? DepartmentId { get; set; }

    /// <summary>
    /// 部门名称
    /// </summary>
    public string DepartmentName { get; set; }

    /// <summary>
    /// 用户类型
    /// </summary>
    public UserTypeEnum UserType { get; set; }

    /// <summary>
    /// 状态
    /// </summary>
    public CommonStatusEnum Status { get; set; }
}
